contributing.md

```diff
@@ -23,3 +23,10 @@ Don't worry if it seems like there are a lot of changes requested, or if they se
 If you need any help fulfilling a change request, just add a comment and any one of the maintainers can help you out (this is why step 8 is important)
 10. After any changes have been completed, your pull request will be merged, and you will see your changes/new files on the official repository
 11. Pat yourself on the back for sharing your knowledge with the community!
+  
+  
+### Tips and things to keep in mind
+1. This repository is run mainly by volunteers, with other jobs and families, so your Pull Request may take some time before it's allowed to merge. This will just depend on what other things the maintainers have on at any given time, please don't think anyone is ignoring your contributions!
+2. Please try to keep your Pull Requests to as few a files as possible. For example: if you have made some spelling changes to a main file, and are also uploading a script to the contrib folder, please try to separate these into two different Pull Requests so that the workload for the maintainers is more manageable. Keeping each PR smaller will also mean that it will be merged much faster!
+3. Comments or reviews on your Pull Requests are *never* intended to be negative, however if you are unused to the matter of fact way that Git reviews are completed it may seem that way. If you believe someone is being genuinely antagonistic, please contact the Rory [via email](mailto:contact@allensynthesis.co.uk) or [on Discord](https://discordapp.com/users/roryjamesallen#6370).
+4. If you need any help with any stage of this process (or anything else), please don't hesistate to ask questions in the [Discussions page](https://github.com/Allen-Synthesis/EuroPi/discussions) or email Rory directly at [contact@allensynthesis.co.uk](mailto:contact@allensynthesis.co.uk).
```

Comment:
I never even noticed that this discussion page existed! Thanks for pointing it out. I'll check on it periodically. 

That said, more discussion seems to happen on discord than here on github, so you may want to mention that here as well.

Reply:
Very good point, I missed a discussion on here recently because it gets so little use, so I'm tempted to make the Discord the primary place for that and just replace the link to GitHub Discussions